tests/wpunit/AssertValidSchemaTest.php

```diff
@@ -350,6 +350,78 @@ public function testSchemaCallsLazyLoadingTypesWhenNeeded() {
 		$this->assertSame( 'bar', $actual['data']['example']['foo'] );
 
 		$expected_filter_calls = [ 'graphql_wp_object_type_config', 'graphql_object_fields', 'graphql_get_type' ];
-		$this->assertSame( $expected_filter_calls, $filter_calls );
+		foreach ( $expected_filter_calls as $expected_filter_call ) {
+			$this->assertTrue( in_array( $expected_filter_call, $filter_calls, true ) );
+		}
+
+		$this->assertSame( count( $expected_filter_calls ), count( $filter_calls ) );
+	}
```

Comment:
How is this different than the `assertSame`?

Reply:
Tests for me were returning them in different order. Always the same 3, but sometimes in different order. 

Perhaps that's something we should investigate more deeply instead of change this test